it/src/test/java/org/corfudb/universe/scenario/ScenarioIT.java

```diff
@@ -0,0 +1,57 @@
+package org.corfudb.universe.scenario;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import org.corfudb.universe.Universe;
+import org.corfudb.universe.cluster.Cluster.ClusterParams;
+import org.corfudb.universe.cluster.docker.DockerCluster;
+import org.corfudb.universe.scenario.config.ScenarioConfig;
+import org.corfudb.universe.scenario.config.ScenarioConfigParser;
+import org.corfudb.universe.scenario.fixture.Fixtures.ClusterFixture;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.List;
+
+public class ScenarioIT {
+    private static final Universe UNIVERSE = Universe.getInstance();
+
+
+    private DockerCluster dockerCluster;
+    private final DockerClient docker;
+
+    public ScenarioIT() throws Exception {
+        this.docker = DefaultDockerClient.fromEnv().build();
+    }
+
+    @After
+    public void tearDown() {
+        if (dockerCluster != null) {
+            dockerCluster.shutdown();
+        }
+    }
+
+    @Test
+    public void scenarioTest() throws Exception {
+        ClusterFixture clusterFixture = ClusterFixture.builder().build();
```

Comment:
When running this scenario I'm not sure I understand. This is a cluster fixture which is a 3 node cluster setup. But still upon initialization only one server is instantiated and the other two derive from actions configured in the json. If it is a 3-Node Cluster shouldn't it instantiate a 3-node cluster plus any actions... 

In addition if I add an action to add a forth node it fails, why?

Reply:
why do we need to set the number of nodes in the fixture in advance? this would give us less flexibility to add/remove nodes as our own custom scenarios.

Reply:
Update: from now we can add/remove nodes independently. You can create a cluster with an initial size, then nodes can be added in the test dynamically.